pkg/service/roommanager.go

```diff
@@ -527,6 +527,25 @@ func (r *RoomManager) handleRTCMessage(roomName, identity string, msg *livekit.R
 				"tracks", rm.UpdateSubscriptions.TrackSids,
 				"subscribe", rm.UpdateSubscriptions.Subscribe)
 		}
+	case *livekit.RTCNodeMessage_SendData:
+		logger.Debugw("SendData", "message", rm)
+
+		kind := livekit.DataPacket_RELIABLE // Default to RELIABLE
+		if rm.SendData.Kind == "lossy" {
```

Comment:
another question here, maybe related to the protocol.. but is there a reason why SendData.Kind is not a `DataPacket_Kind` ? Ideally we would use the same protobuf types, instead of comparing against strings.

Reply:
I will clean up that part, but is the Json to protobuf conversion for enums case in-sensitive ?
I will try later✌🏻

I wanted to support cases such as
```
{
  "kind": "lossy",
  "data": ......
}
```

Reply:
I believe it does!